### PR TITLE
feat(polls): add option to import from csv

### PR DESCRIPTION
**Pull request by NotFish232**
## Proposed changes
- Add option when creating polls to import from a csv

## Brief description of rationale
- Inputting election data is tedious and should be automated

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,6 +20,8 @@ dump.rdb
 *.log
 *.swp
 celerybeat-schedule
+celerybeat-schedule.dir
```
Comment (alanzhu0) [inline .gitignore:23]: Why are these two lines added?

File: intranet/static/js/polls.js
Comment (alanzhu0) [inline intranet/static/js/polls.js:1]: Please remove the changes by an auto-formatter and include only new code.

File: intranet/static/js/vendor/jquery.csv.js
Comment (alanzhu0) [inline intranet/static/js/vendor/jquery.csv.js:1]: Use a minified file if possible.

File: intranet/templates/polls/vote.html
Comment (alanzhu0) [inline intranet/templates/polls/vote.html:1]: Please remove this change.

**Review by alanzhu0 — CHANGES_REQUESTED** [review]
See comments. Small changes. Functionality looks good.

File: .gitignore (hunk 1 above)
Comment (NotFish232) [reply .gitignore:23]: Docker dev env spawns these files for me

**Review by alanzhu0 — APPROVED** [review]
LGTM

**Pull request merged**